src/Compilers/Core/CodeAnalysisTest/MetadataReferences/ModuleMetadataTests.cs

```diff
@@ -57,8 +59,10 @@ public unsafe void CreateFromMetadata_Module()
         public unsafe void CreateFromImage()
         {
             Assert.Throws<ArgumentNullException>(() => ModuleMetadata.CreateFromImage(IntPtr.Zero, 0));
-            Assert.Throws<ArgumentOutOfRangeException>(() => { fixed (byte* ptr = new byte[] { 1, 2, 3 }) ModuleMetadata.CreateFromImage((IntPtr)ptr, 0); });
-            Assert.Throws<ArgumentOutOfRangeException>(() => { fixed (byte* ptr = new byte[] { 1, 2, 3 }) ModuleMetadata.CreateFromImage((IntPtr)ptr, -1); });
+            Assert.Throws<ArgumentOutOfRangeException>(() => { fixed (byte* ptr = new byte[] { 1, 2, 3 })
+                    ModuleMetadata.CreateFromImage((IntPtr)ptr, 0); });
+            Assert.Throws<ArgumentOutOfRangeException>(() => { fixed (byte* ptr = new byte[] { 1, 2, 3 })
+                    ModuleMetadata.CreateFromImage((IntPtr)ptr, -1); });
```

Comment:
📝 This one has a weird fix. I think the case can be narrowed down to the fact that `fixed` isn't the first token on its line.

Reply:
these ones should probably be forced to change.  i think a single-line lambda is fine with a single (non-nested) statement in it.  But if you have complex statements like this, it should likely be forced to multiple lines.

Reply:
I agree that a change is warranted, but I lean towards this as being the ideal form:

```csharp
Assert.Throws<ArgumentOutOfRangeException>(
    () =>
    {
        fixed (byte* ptr = new byte[] { 1, 2, 3 })
            ModuleMetadata.CreateFromImage((IntPtr)ptr, -1);
    });
```

Or if the `() =>` doesn't change lines, this:

```csharp
Assert.Throws<ArgumentOutOfRangeException>(() =>
{
    fixed (byte* ptr = new byte[] { 1, 2, 3 })
        ModuleMetadata.CreateFromImage((IntPtr)ptr, -1);
});
```

Reply:
i produce the latter